zerver/lib/digest.py

```diff
@@ -378,6 +403,10 @@ def bulk_get_digest_context(
         context["new_channels"] = new_streams
```

Comment:
For the message content disabled case, I don't think we need to pass this context parameter.